### DISCUSSION TITLE
How do I add ancestor's title to table?

### DISCUSSION THREAD

**Discussion by jwhonce** [mabeyj/trilium-collection-views#38] [Q&A] [answered]
- Collection View renderNote
  - Category
     - tagged Note

I would like a table where: 
| Title | Category Title | tag |  

I have configured a table that shows the tagged Note's title and the value of tag. But I have yet to find the magic to include the Category. Something like `#attribute="note.ancestors.title,header=Category". Is this possible?

**Comment by mabeyj** [comment, marked as answer]
This can be done by adding a `category` relation to Tagged Note which points to the parent Category note, and then `#attribute="category,header=Category"` should display the category name.

Without adding that explicit relation, this is currently not possible since there is no way to refer to a note's parent/ancestor. I will add a feature request for this.
